tree-sitter-dscexpression/grammar.js

```diff
@@ -29,7 +29,7 @@ module.exports = grammar({
     _quotedString: $ => seq('\'', $.string, '\''),
     // ARM strings do not allow to contain single-quote characters
     string: $ => /[^']*/,
-    number: $ => /-?\d+/,
+    number: $ => /[+-]?([0-9]*[.])?[0-9]+/,
```

Comment:
Do we need to support `+1`?  Do we know if ARM allows a `+` symbol?

Why use `[0-9]` instead of `\d`?  Do we want to support `.1` vs `0.1`?

Reply:
updated to continue using `\d` - will not support `.1` or `+`